### ISSUE TITLE
how to overfit train for a single image

### ISSUE THREAD

**Issue by krishnak** [qqwweee/keras-yolo3#633]
I want to overfit a YOLO_v3 model to a single image to test the loss function.

I am using a fixed input of 240x320 instead of image augementation to simplify few things. I am detecting only 1 class. I am training from scratch.

My question is if my single image has only 2 bounding boxes. When I run Kmeans on that image annotation, I only get 2 anchors. What do I do for fulfilling the 9 anchor criteria?

Or is there a better way to implement what I am trying to implement
